WooCommerce/Classes/ViewRelated/Orders/OrdersTabbedViewController.swift

```diff
@@ -74,7 +74,9 @@ final class OrdersTabbedViewController: ButtonBarPagerTabStripViewController {
     /// Shows Order Creation flow.
     ///
     @objc private func createOrder() {
```

Comment:
Following the name convention of the other methods, I'd say to rename it on 
```suggestion
    @objc private func displayCreateOrder() {
```